cpp/tests/community/louvain_mg_test.cpp

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
I recommend renaming this to mg_louvain_test.cpp as the executable name is MG_LOUVAIN_TEST.

Reply:
Fixed